ext/native/file/file_util.cpp

```diff
@@ -85,13 +83,23 @@ uint64_t GetSize(FILE *f)
 	}
 	return size;
 #else
+#ifdef _WIN32
+	uint64_t pos = _ftelli64(f);
+#else
 	uint64_t pos = ftello(f);
+#endif
 	if (fseek(f, 0, SEEK_END) != 0) {
 		return 0;
 	}
+#ifdef _WIN32
+	uint64_t size = _ftelli64(f);
```

Comment:
Hmm, what are we gaining here by duplicating the code and making it so there's an extra place to miss when fixing bugs?

Not saying this is bad, just "more mingw patches" isn't very descriptive.  6 months from now I may see this code and "fix it", because DRY.

-[Unknown]

Reply:
It's necessary to work around an error. The macro will clash with the mingw headers that provide prototypes for _ftelli64/_fseeki64, etc.

Reply:
Well, at least comment it appropriately.